gapis/trace/android/trace.go

```diff
@@ -65,7 +66,7 @@ const (
 	durationMs                              = 7000
 	gpuCountersDataSourceDescriptorName     = "gpu.counters"
 	gpuRenderStagesDataSourceDescriptorName = "gpu.renderstages"
-	minimumSupportedApiLevel                = 29
+	minimumSupportedApiLevel                = 19
```

Comment:
Are you sure about that? I don't think we support Perfetto before Q.

Reply:
I was using this to test, UI stuff -- didn't mean to check this in